packages/binaryen/project.bri

```diff
@@ -0,0 +1,55 @@
+import * as std from "std";
+import { cmakeBuild } from "cmake";
+import git from "git";
+
+export const project = {
+  name: "binaryen",
+  version: "123",
```

Comment:
Huh, so they just use a flat number for the version!

Reply:
Yeah, I was thinking at the same thing ! I was a bit surprised when I saw that.